templates/data/mongodb/support.html

```diff
@@ -0,0 +1,429 @@
+      <div class="col">
+        <h2>MongoDB support</h2>
+      </div>
+    </div>
+    <div class="u-fixed-width">
+      <div class="p-image-wrapper u-hide--small">
+        <div class="p-image-container is-highlighted">
+          {{ image(url="https://assets.ubuntu.com/v1/6f9200b9-1.png",
+                    alt="",
+                    width="1100",
+                    height="611",
+                    hi_def=True,
+                    attrs={"class": "p-image-container__image"},
+                    loading="lazy") | safe
+          }}
+        </div>
+        <hr class="is-muted" />
+      </div>
+    </div>
+    <div class="p-equal-height-row p-section--shallow">
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Technical support</h3>
+        </div>
+        <p class="p-equal-height-row__item">
+          Your data, our expertise. Included in the Ubuntu Pro + Support pricing is phone and ticket support for MongoDB. Canonical uses the follow-the-sun support model to ensure 24/7 coverage so that your issues are resolved quickly.
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Full-stack support</h3>
+        </div>
+        <p class="p-equal-height-row__item">
+          As the publisher of Ubuntu, Canonical uniquely supports the full stack. Complex issues in enterprise data platforms require analysis and fixes all the way down the stack. Get a single solution for best in class support &mdash; with no loose ends.
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">10 years of security</h3>
+        </div>
+        <p class="p-equal-height-row__item">
+          Not every organisation is willing or able to upgrade their critical data infrastructure on a regular basis. With Canonical, your service stays secure for up to 10 years with no need for major upgrades.
+        </p>
+      </div>
+      <div class="p-equal-height-row__col">
+        <div class="p-equal-height-row__item">
+          <h3 class="p-heading--5">Fast response time</h3>
+        </div>
+        <p class="p-equal-height-row__item">
+          We guarantee 1 hour response time for the most critical issues. Our expertise underpins your SLA.
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <hr class="is-muted" />
+      <div class="col-6 col-start-large-7 col-medium-3 col-start-medium-4">
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="data-mongodb-modal">
+          Get
+        support</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
+      <hr />
+      <div class="col">
+        <h2>Managed services</h2>
+      </div>
+    </div>
```

Comment:
Why a 50-50 row that is not intended to be used as such? The same can be achieved like so: 
```suggestion
    <div class="u-fixed-width p-section--shallow">
      <hr />
      <h2>Managed services</h2>
    </div>
```